Storage/SQL/BrowserDB.swift

```diff
@@ -32,6 +32,14 @@ open class BrowserDB {
         self.db = SwiftData(filename: file, key: secretKey, prevKey: nil, schema: schema, files: files)
     }
 
+    // Remove the DB op from the queue (by marking it cancelled), and if it is already running tell sqlite to cancel it.
+    public func cancel(databaseOperation: Cancellable) {
```

Comment:
Is there any value in making the input here `weak`?

Reply:
Right, there is none.